landbosse/excelio/XlsxReader.py

```diff
@@ -537,6 +544,53 @@ def map_labor_rates(row):
         rsmeans.drop(columns=['Rate USD per unit'], inplace=True)
         rsmeans['Rate USD per unit'] = rsmeans_new_labor_rates
 
+    def apply_cost_and_scaling_modifications_to_project_parameters(self, project_parameters):
+        """
+        This applies the cost and scaling modification to project parameters
+        specified in the project list.
+
+        Note: It is meant to be called on the parameter list AFTER it has been
+        modified with the parametrics by modify_project_data_and_project_list()
+        above.
+
+        It modifies the parameters IN PLACE.
+
+        Parameters
+        ----------
+        project_parameters : pd.Series
+            The project parameters to be modified.
+        """
+        project_size_MW = project_parameters['Number of turbines'] * project_parameters['Turbine rating MW']
+        hub_height_m = project_parameters['Hub height m']
+        flag_use_user_homerun = project_parameters['Flag for user - defined home run trench length (0 = no; 1 = yes)']
+        nameplate = project_parameters['Turbing rating MW']
+
+        distance_to_interconnect_mi = 0.0 if project_size_MW <= 20 else (0.009375 * project_size_MW + 0.625)
+        interconnect_voltage_kV = 0.4398 * project_size_MW + 60.204
+        new_switchyard_y_n = 'n' if project_size_MW <= 40 else 'y'
+        road_length_adder_m = 1e3 if project_size_MW <= 20 else (13.542 * project_size_MW + 1458.3)
+        breakpoint_between_base_and_topping = 0.0 if project_size_MW <= 20 else (35 / hub_height_m)  # if greater than 20 MW, then breakpoint between base and topping at 35 meters
+        number_of_access_roads = 0.0 if project_size_MW <= 20 else ceil(0.0052 * project_size_MW + 0.7917)
+        number_of_highway_permits = ceil(0.2 * project_parameters['Number of turbines'])
+        if flag_use_user_homerun is 1:
+             user_input_homerun_km = 0.1776 * project_size_MW - 2.551
+        rate_deliveries = ceil(15 / nameplate)  # 10 deliveries per week for 1.5 MW machines
+
```

Comment:
It looked like there were a few parameters missing here. Let me know if I'm mistaken. I added user_input_homerun_km and rate_deliveries. There were also a few bugs with how the equations were implemented. Road length = 1e3, not 1e4 if project size is below 20 MW, and number of access roads equations should use 0.0052 rather than 0.052.

Reply:
You're right. Sorry about the oversight!